serialisation/src/test/java/org/ehrbase/serialisation/flatencoding/std/umarshal/FlatJsonUnmarshallerTest.java

```diff
@@ -64,12 +64,6 @@ public void unmarshal() throws IOException, XmlException {
         "2020-05-11T22:53:12.039139+02:00");
     assertThat(observation.getSubject()).isNotNull();
     assertThat(observation.getSubject().getClass()).isEqualTo(PartySelf.class);
-
```

Comment:
Please add the new compostionValidate where ever you removed the old one.

Reply:
The validation of these FLAT compositions is tested in `FlatValidationTest`